Kata/Classes/KataCalculator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kata
+{
+    public class KataCalculator
+    {
+
+        public Product Product;
+       public TaxCalculator taxCalculator;
```

Comment:
I don't think any of these prosperities need to be public, lets convert to private